cla_public/static-templates/errors/50x.html

```diff
@@ -121,51 +122,51 @@
           </p>
-          <p class="govuk-body">Textphone:<br />
+          <p class="govuk-body">{% trans %}Textphone:{% endtrans %}<br/>
             <strong class="govuk-!-font-weight-bold">0845 609 6677</strong>
           </p>
           <p class="govuk-body">
-            <a href="https://www.gov.uk/call-charges">Find out about call charges here</a>.
+            <a href="https://www.gov.uk/call-charges">{% trans %}Find out about call charges here{% endtrans %}</a>.
           </p>
-          <p class="govuk-body">Opening times:<br />
-            <strong class="govuk-!-font-weight-bold">Monday to Friday: 9am to 8pm<br />Saturday: 9am to 12.30pm</strong>
+          <p class="govuk-body">{% trans %}Opening times:{% endtrans %}<br />
+            <strong class="govuk-!-font-weight-bold">{% trans %}Monday to Friday: 9am to 8pm{% endtrans %}<br />{% trans %}Saturday: 9am to 12.30pm{% endtrans %}</strong>
           </p>
-          <div class="govuk-inset-text">
-            This service is only available if you are a resident of England or Wales.
+          <div class="govuk-inset-text">{% trans %}
+            This service is only available if you are a resident of England or Wales.{% endtrans %}
           </div>
         </main>
       </div>
     <footer class="govuk-footer " role="contentinfo">
       <div class="govuk-width-container ">
         <div class="govuk-footer__meta">
           <div class="govuk-footer__meta-item govuk-footer__meta-item--grow">
-            <h2 class="govuk-visually-hidden">Support links</h2>
+            <h2 class="govuk-visually-hidden">{% trans %}Support links{% endtrans %}</h2>
             <ul class="govuk-footer__inline-list">
               <li class="govuk-footer__inline-list-item">
-                <a class="govuk-footer__link"  href="/privacy">Privacy Policy</a>
+                <a class="govuk-footer__link"  href="/privacy">{% trans %}Privacy Policy{% endtrans %}</a>
               </li>
               <li class="govuk-footer__inline-list-item">
-                <a class="govuk-footer__link"  href="/cookies">Cookies</a>
+                <a class="govuk-footer__link"  href="/cookies">{% trans %}Cookies{% endtrans %}</a>
               </li>
               <li class="govuk-footer__inline-list-item">
-                <a class="govuk-footer__link"  href="/reasons-for-contacting">Contact us</a>
+                <a class="govuk-footer__link"  href="/reasons-for-contacting">{% trans %}Contact us{% endtrans %}</a>
               </li>
               <li class="govuk-footer__inline-list-item">
-                <a class="govuk-footer__link"  class="govuk-footer__link" href="https://mojdigital.blog.gov.uk/"><abbr title="Ministry of Justice">MoJ</abbr> Digital Services</a>
+                <a class="govuk-footer__link"  class="govuk-footer__link" href="https://mojdigital.blog.gov.uk/"><abbr title="Ministry of Justice">{% trans %}MoJ</abbr> Digital Services% endtrans %}</a>
```

Comment:
Is that a missing { here?

I think the Welsh translation should include the abbr tag - so we can translate the "Ministry of Justice" bit as well.

Or, maybe separated:
`<a class="govuk-footer__link"  class="govuk-footer__link" href="https://mojdigital.blog.gov.uk/"><abbr title="{% trans %}Ministry of Justice{% endtrans %}">{% trans %}MoJ</abbr> Digital Services{% endtrans %}</a>`

Reply:
@MalcolmVonMoJ - thanks, good spot. Will move the abbr tag into the translation.